lib/apiservers/engine/backends/volume.go

```diff
@@ -53,6 +53,11 @@ var validDriverOptsKeys = map[string]struct{}{
 	DriverArgImageKey:     {},
 }
 
+var supportedVolDrivers = map[string]struct{}{
+	"local":   struct{}{},
```

Comment:
@hickeng @matthewavery I've changed the code to only display `vsphere` in `docker info`. If we want to display either `vsphere` or `local`, I think that displaying the former will help us differentiate between a VCH and other hosts in Admiral that support the `local` driver.